Makefile

```diff
@@ -77,9 +81,17 @@ black:
 pyupgrade:
 	"${DC}" run --rm webapp pyupgrade --exit-zero-even-if-changed --py38-plus *.py `find pontoon -name \*.py`
 
+pyupgrade-check:
+	"${DC}" run --rm webapp pyupgrade --py38-plus *.py `find pontoon -name \*.py`
+
 types:
 	"${DC}" run --rm -w //app/frontend webapp yarn types
 
+black-check:
+	"${DC}" run --rm webapp black --check pontoon
+
+lint: eslint types flake8 black-check pyupgrade-check
```

Comment:
Should `check-prettier` be part of the list?

Let's use `check` consistently, either as a prefix of postfix.